### PR TITLE
Adding in a post proc module that only does Btagging and TopTaggin up…

### PR DESCRIPTION
**Pull request by scarletnorberg**
…dating the allInOne btagEff file to use post processed instead of pre updating condor file to include the Stop0l_postbTag.py and be v7

### CODE REVIEW

**Comment by zhenbinwu** [comment]
Can you just tag the v6p5 branch, and update Stop0l_postpost.py  to include btagSF? I want to avoid too many duplicate files. If we just updating the systematic, we don't need to rerun baseline right?

**Comment by scarletnorberg** [comment]
Just for clarification "Can you just tag the v6p5 branch" Are you asking me to make a branch that is dev_v6p5? Are you asking me to do a PR to dev_v6p5? I think master is the most up to date and it includes the PR from Joe with toptagging. Everything else is done I just want not quiet sure what you meant by this first part.

**Comment by zhenbinwu** [comment]
Branch is different from tag. https://en.wikibooks.org/wiki/Git/Advanced#Tags_vs_Branches
tag meaning create a tag

**Comment by scarletnorberg** [comment]
True sorry for my confusion. I wanted to merge this in master and then I figured I would tag master.

File: python/processors/Stop0l_postpost.py [hunk 1]
```diff
@@ -194,6 +195,8 @@ def main(args):
     if not isdata:
         ## Major modules for MC
         mods += [
+            btagSFProducer(args.era, algo="deepcsv"),
+            BtagSFWeightProducer(DataDepInputs[dataType][args.era]["bTagEff"], args.sampleName, DeepCSVMediumWP[args.era], isfastsim=isfastsim),
             TopReweightProducer(args.era, args.sampleName, isData=isdata),
             DeepTopProducer(args.era, taggerWorkingDirectory, "JESUp", sampleName=args.sampleName, isFastSim=isfastsim, isData=isdata),
```
Comment (zhenbinwu) [inline python/processors/Stop0l_postpost.py:201]: If we don't expect change from other modules, we don't have to rerun them.

File: python/processors/Condor/SubmitLPC.py [hunk 2]
```diff
@@ -17,10 +17,10 @@
 from multiprocessing import Pool
 from itertools import izip_longest
 
-DelExe    = '../Stop0l_postpost.py'
+DelExe    = '../Stop0l_postbTag.py'
```
Comment (zhenbinwu) [inline python/processors/Condor/SubmitLPC.py:20]: This is still wrong

**Comment by scarletnorberg** [comment]
Sure I can tag this as dev6p5 that is fine.

File: python/processors/Condor/SubmitLPC.py (hunk 2 above)
Comment (scarletnorberg) [reply python/processors/Condor/SubmitLPC.py:20]: I will make that change sorry I forgot I had change that thanks for the check.

Comment (scarletnorberg) [reply python/processors/Condor/SubmitLPC.py:20]: I had already changed it I just did not do git add

Comment (scarletnorberg) [reply python/processors/Condor/SubmitLPC.py:20]: Locally I saw the code updated and was confused for a second. Why.

**Comment by scarletnorberg** [comment]
Finally postpost is what is written and I tested it but once everything is pushed to master I will re test it tag it and we should be ready to go again and run the fullsim.

**Comment by scarletnorberg** [comment]
Please have a look @zhenbinwu

File: python/processors/Stop0l_postpost.py (hunk 1 above)
Comment (pastika) [reply python/processors/Stop0l_postpost.py:201]: I think we can remove all DeepTopProducer modules as we only need to redo the merged top uncertainties which happens in "SoftBDeepAK8SFProducer".  We can also remove "TopReweightProducer"?

**Comment by zhenbinwu** [comment]
What about the 2018 JEC update? Do you want to rerun with the wrong file? It is not in this PR.

File: python/processors/Stop0l_postpost.py (hunk 1 above)
Comment (scarletnorberg) [reply python/processors/Stop0l_postpost.py:201]: So we only need DeepTopProducer then? I can work on only running that just want to confirm.

**Comment by scarletnorberg** [comment]
We already have the update for the fullsim and I am double check the fastsim for 2018 but sense all of fullsim meaning 16/17/18 need to be rerun I thought better to start that Friday. As you can see from the files changed: allInOne_bTagEff_deepCSVb_med.root  this is the first file listed because it is updated already running on post instead of pre.

**Comment by zhenbinwu** [comment]
In that file, only 18 is updated or all 3 years are updated?

File: python/processors/Stop0l_postpost.py (hunk 1 above)
Comment (zhenbinwu) [reply python/processors/Stop0l_postpost.py:201]: If both are updating on the systematic, is it safe to run just these two modules?

**Comment by scarletnorberg** [comment]
All three years are run on post just to be sure.

File: python/processors/Stop0l_postpost.py (hunk 1 above)
Comment (scarletnorberg) [reply python/processors/Stop0l_postpost.py]: I updated this technically it runs but events are still being produce on my test file. Please check that the correct top tagger modules are being run. When the file is done I will ping @pastika to have a look at the top variables that we expect to be changed. I will check the btagging to make sure they get updated.

Comment (zhenbinwu) [reply python/processors/Stop0l_postpost.py]: Ok, I suspect the TopReweightProducer and baseline won't need to rerun. But please check.

**Pull request merged**